### PR TITLE
(#25) Term name - no pronunciation

### LINKED ISSUE #25: Terms without pronunciation don't appear in the dictionary box
### Issue description
If you do a search for "NCI" or "NIH" on react-app-dev, the definition box does not display the term (but does display the definition).

> **ESTIMATE** 2

### Steps to reproduce the issue

1.  Go to react-app-dev for sitewide search on develop and put in NIH as the swKeyword
2. Note that a definition is returned, but the term does not appear in the Definition box at the top below "Definition"
3. See images below

This is what is expected:
image

This is incorrect:
image

### What's the expected result?
- The term appears below "Definition"

### What's the actual result?
- The term does not appear anywhere in the definition box

### Additional details / screenshot
- Screenshot

### Related Tickets
* Issue #9999
* Issue #9999

### PR DESCRIPTION
**Pull request by seyilonge-nci**
Closes #25

* Display term name when no pronunciation exists for definition

### CODE REVIEW

**Comment by seyilonge-nci** [comment]
@VictoriaSunNIH 
Please review: https://react-app-dev.cancer.gov/sitewide-search-app/pr-27?cfg=0&swkeyword=nci

File: src/components/molecules/definition/definition.jsx [hunk 1]
```diff
@@ -38,15 +38,11 @@ const Definition = ({ results }) => {
```
Comment (arcepaul) [inline src/components/molecules/definition/definition.jsx:38]: no need for this to be in a render function anymore

Comment (seyilonge-nci) [reply src/components/molecules/definition/definition.jsx:38]: True! Will update.

Comment (seyilonge-nci) [reply src/components/molecules/definition/definition.jsx:38]: @arcepaul Done!

**Pull request merged**